src/content/guide/how-to-build-a-product/authentication.md

```diff
@@ -0,0 +1,423 @@
+---
+title: Authentication & Security
+columns: two
+template: guide.hbs
+order: 4
+---
+
+# {{title}}
+
+## Introduction
+
+For any Particle-powered product to function properly, there are **four involved
+parties**:
+
+* Particle
+* Your organization's applications
+* A device
+* The customer
+
+![four involved authentication parties](/assets/images/four-involved-parties.png)
+<p class="caption">Each of the four parties plays a critical role in how
+your product will function</p>
+
+Understanding how all four interact, manage data, and  secure information is critical to launching a successful product on the Particle platform.
+Specifically, this section of the guide will go deep into decisions you
+must make on authentication and security with regards to your product.
+The results of these decisions will impact where and how data is
+stored as well as the ways in which your organization's applications
+interact with Particle to control devices and manage customers.
+
+
+## OAuth
+Particle tightly adheres to OAuth specifications to ensure secure and
+correct access to private data and devices. OAuth is an open standard
+for authorization, providing client applications "secure delegated
+access" to server resources on behalf of a resource owner
+([Wikipedia](https://en.wikipedia.org/wiki/OAuth)).
+
+Integral to OAuth is the concept of *clients*, and *client credentials*.
+An OAuth client generally represents an application, like a native iOS
+application running on an iPhone or a web app running in a browser. As
+part of the setup process for your product, you will need to create an
```

Comment:
you will need to create *one or more* OAuth *clients* for your organization.

Or "at least one" or something like that...